packages/vuetify/src/locale/no.ts

```diff
@@ -31,15 +31,15 @@ export default {
   },
   datePicker: {
     ok: 'OK',
-    cancel: 'Cancel',
+    cancel: 'Avbryt',
     range: {
-      title: 'Select dates',
-      header: 'Enter dates',
+      title: 'Velg datoer',
+      header: 'Velg datoer',
     },
-    title: 'Select date',
-    header: 'Enter date',
+    title: 'Velg dato',
+    header: 'Velg dato',
     input: {
-      placeholder: 'Enter date',
+      placeholder: 'Velg dato',
```

Comment:
Maybe the context where this is used suggests otherwise, but could "Fyll inn dato" be a better translation for "Enter date"? That would differentiate between "enter" and "select", and might make sense for a placeholder at least.

Reply:
![image](https://github.com/vuetifyjs/vuetify/assets/1843267/44afd197-7980-4706-8e37-68a62e17f175)

This is where the title and header is used. I feel like it "Fyll inn dato" doesnt work here. 

Here is with input-mode="keyboard", and here I agree, it makes more sense with "Fyll inn dato":
![image](https://github.com/vuetifyjs/vuetify/assets/1843267/e8f0c49b-87f0-40ea-8b37-6b6a83a3ecb5)